src/static/css/page.css

```
@@ -276,15 +276,28 @@
 }
 
 .authors .social {
-  display: inline-block;
-  vertical-align: middle;
+  font-size: 0.9em;
 }
 
 .authors .social a {
-  display: inline-block;
-  vertical-align: middle;
-  margin: 0 5px;
-  margin: 0 0.3125rem;
+  margin: 0 10px 0 0;
+  margin: 0 0.625rem 0 0;
```

Could this be simplified to `margin-right` or are the 0s significant?
```suggestion
  margin-right: 10px;
  margin-right: 0.625rem;
```

---

I also wonder if it's worth doing this to maximize space before wrapping.

```css
.authors .social a:last-child {
  margin-right: 0;
}
```

---

Added both suggestions!